vscode-pets-integration/experiment1/scripts/sprite-generation/generate_forkmonkey.py

```
@@ -0,0 +1,240 @@
+            
+            # Save original
+            base_path = os.path.join(OUTPUT_DIR, "base_character.png")
+            image.save(base_path)
+            print(f"✅ Base character saved: {base_path}")
+            print(f"   Size: {image.size}, Mode: {image.mode}")
+            
+            return image
+    
+    raise Exception("No image generated")
+
+def generate_animation_frame(client, animation_name, frame_number, total_frames, reference_image=None):
```

Ruff (0.14.10)

Unused function argument: `reference_image`

(ARG001)